Documentation/DemoApps/GRDBCombineDemo/GRDBCombineDemoUITests/GRDBCombineDemoUITests.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  GRDBCombineDemoUITests.swift
+//  GRDBCombineDemoUITests
+//
+//  Created by Peter Steinberger on 08.04.21.
+//
+
+import XCTest
+
+class GRDBCombineDemoUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+    }
+
+    override func tearDownWithError() throws {
+    }
+
+    func runApp() -> XCUIApplication {
+        // UI tests must launch the application that they test.
+        let app = XCUIApplication()
+        // We enforce a fresh database to ensure 8 random players.
+        app.launchArguments = ["-reset", "-fixedTestData"]
+        app.launch()
+        return app
+    }
+
+    func testInitialSortingIsByScore() throws {
+        let app = runApp()
+
+        let getFirstCell = { () -> XCUIElement in
+            app.tables.element(boundBy: 0).cells.element(boundBy: 0)
+        }
+
+        // Given default sorting, Henriette must be on top
+        XCTAssertEqual(getFirstCell().label, "Craig, 8 points")
+
+        // Reverse sorting will bring Arthur on top (sort by name)
+        app.buttons["Score"].tap()
+        XCTAssertEqual(getFirstCell().label, "Arthur, 5 points")
+    }
+
+    func testListAndAddPlayer() throws {
```

Comment:
This test is unrelated and just tests basic add functionality.

Reply:
It's quite educational, perfect for a demo app! 👍